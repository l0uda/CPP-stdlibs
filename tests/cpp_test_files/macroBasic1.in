#define A Preprocesor
#define B funguje
#define C dob≈ôe
A B C