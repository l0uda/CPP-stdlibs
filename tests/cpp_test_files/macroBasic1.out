original:

#define A Preprocesor
#define B funguje
#define C dobře
A B C
-----------
preprocessed:

Preprocesor funguje dobře
